packages/fxa-content-server/tests/functional/post_verify/force_password_change.js

```diff
@@ -109,48 +104,5 @@ registerSuite('post_verify_force_password_change', {
         .then(fillOutForceChangePassword(PASSWORD, NEW_PASSWORD))
         .then(testAtOAuthApp());
     },
-    'force change password on login - w/TOTP': function () {
-      const self = this.remote;
-      return this.remote
-        .then(openPage(ENTER_EMAIL_SYNC_URL, selectors.ENTER_EMAIL.HEADER))
-        .then(fillOutEmailFirstSignIn(email, PASSWORD))
-        .then(fillOutSignInTokenCode(email, 0))
-
-        .then(fillOutForceChangePassword(PASSWORD, NEW_PASSWORD))
-
-        .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER))
-
-        .then(enableTotp())
-        .then((_secret) => {
-          secret = _secret;
-        })
-        .then(
-          clearBrowserState({
-            '123done': true,
-            contentServer: true,
-            force: true,
-          })
-        )
-        .then(openPage(ENTER_EMAIL_SYNC_URL, selectors.ENTER_EMAIL.HEADER))
-        .then(fillOutEmailFirstSignIn(email, NEW_PASSWORD))
-
-        .then(testElementExists(selectors.TOTP_SIGNIN.HEADER))
-
-        .then(() => {
-          return self.then(
-            type(selectors.TOTP_SIGNIN.INPUT, generateTotpCode(secret))
-          );
-        })
-        .then(click(selectors.TOTP_SIGNIN.SUBMIT))
-
-        .then(testIsBrowserNotified('fxaccounts:login'))
-
-        .then(
-          testElementExists(selectors.POST_VERIFY_FORCE_PASSWORD_CHANGE.HEADER)
-        )
-
-        .then(fillOutForceChangePassword(NEW_PASSWORD, PASSWORD))
-        .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER));
-    },
```

Comment:
Confirmed with Ankita that all of these tests will be removed soon since these are the old Intern-based tests, and that it's okay to remove this.